src/lib/viewers/media/DashViewer.js

```diff
@@ -104,6 +107,8 @@ class DashViewer extends VideoBaseViewer {
      * @return {void}
      */
     load() {
+        this.loadUI();
```

Comment:
Moving major calls like this can be a somewhat scary change. Does everything work as expected? Does this get called more or less often than before?